problem_builder/public/css/problem-builder.css

```diff
@@ -0,0 +1,7 @@
+.pb-clarification span.clarification i {
+    font-style: normal;
+}
+
+.pb-clarification span.clarification i:hover {
+    color: rgb(0, 159, 230);
+}
```

Comment:
Can this be combined with `mentoring.css` ? Mentoring.css should really be renamed to `problem_builder.css` since it applies to the block as a whole. This would be a good opportunity to do so, and would reduce the number of separate CSS/JS files we are sending to the browser.

Also nit: missing newline at EOF